R/gitbook.R

```diff
@@ -5,13 +5,13 @@
 #' @inheritParams html_chapters
 #' @param fig_caption,number_sections,self_contained,lib_dir,pandoc_args ...
 #'   Arguments to be passed to \code{rmarkdown::\link{html_document}()}
-#'   (\code{...} not including \code{toc}, \code{theme}, and \code{template}).
+#'   (\code{...} not including \code{toc}, and \code{theme}).
 #' @param config A list of configuration options for the gitbook style, such as
 #'   the font/theme settings.
 #' @export
 gitbook = function(
   fig_caption = TRUE, number_sections = TRUE, self_contained = FALSE,
-  lib_dir = 'libs', pandoc_args = NULL, ...,
+  lib_dir = 'libs', pandoc_args = NULL, ..., template = bookdown_file('templates', 'gitbook.html'),
```

Comment:
I think it is also worth documenting the `template` argument explicitly. In particular, we need to warn users that you have to fully understand the default template before using a custom template for gitbook. I don't know what your use case here. I feel it should be very rare that one needs a custom template for gitbook.

Reply:
I want to use extra pandoc variables to place certain elements at a specific location which specific formatting at the title page to match our corporate identity style. The [template](https://github.com/inbo/INBOmd/tree/cd50301646272571482058265ed20bef3c6e5d4f/inst/rmarkdown/templates/inbo_rapport/skeleton) will be used by people less knowledgeable on bookdown.

I'll make the requested changes. I'll recommend to start from the [gitbook.html template](https://github.com/rstudio/bookdown/blob/master/inst/templates/gitbook.html) when creating a custom template.